docs/5.x/cluster.md

```diff
@@ -2069,3 +2069,50 @@ With `promiscuous-bridge`, the behavior is similar to that of the kubenet networ
 [PodSecurityPolicies]: https://kubernetes.io/docs/concepts/policy/pod-security-policy/
 [RBAC]: https://kubernetes.io/docs/admin/authorization/rbac/
 [promiscuous-mode]: https://en.wikipedia.org/wiki/Promiscuous_mode
+
+## Customizing Cluster DNS
+
+Gravity uses [CoreDNS](https://coredns.io) for DNS resolution and service discovery within the cluster. 
+
+The default configuration can be customized, by creating a kubernetes configmap kube-system/coredns.
+
+```
+kind: ConfigMap
```

Comment:
Does this config map represent default configuration we ship, or just some example configuration? It is not clear and I think should be made clear.